HACKING.md

```diff
@@ -42,73 +53,135 @@ Add `$GOPATH/bin` to your `PATH`, so you can run the go programs you install:
 your `$GOPATH` is more complex than a single entry you'll need to adjust the
 above).
 
-Note that if you are using go 1.16 or newer you need to disable the
-go modules feature. Use:
 
-    export GO111MODULE=off
+### Getting the snapd sources
 
-for this.
+The easiest way to get the source for `snapd` is to clone the Github repository in
+a directory where you have read-write permissions, such as your home directory.
 
+    cd ~/
```

Comment:
`cd` without parameters is enough

Reply:
I never knew this! But,I would suggest it's a little clearer with the tilde as we assume home for the remainder of the text. Also, it's used in another example below.

Reply:
Nah, I'm not protesting. Though by using `cd ~` we miss a good opportunity to educate people about this trick :-)